### DISCUSSION TITLE
Course is not generated

### DISCUSSION THREAD

**Discussion by dnwalker68** [Courseplay/Courseplay_FS22#2766] [Q&A]
Hello,
I have installed the current version of Coursplay and am trying to generate a course for section 6 on the Erlengrat map. Of course, I retracted the field borders manually. No matter what settings I specify in the course profile, no course is calculated. When I start the course generator an animated circle appears and nothing else happens. Even after an eternity of time nothing changes in the situation. My only option is to end the game using the touch manager. Where is the problem? Maybe you have a tip for me
Thanks and regards
Dieter

**Comment by rohne83** [comment]
This means that CP is calculating but doesn't find a feasible solution for creating this course in short time. On very huge and/or weirdly shaped fields that's something that can happen. Either you wait a bit longer (can take several minutes on nasty fields, and cp will tell you then if it still wasn't able to calculate a course), or you edit the field to be a bit more helper friendly or you play around with the course settings.

**Comment by Tensuko** [comment]
Screenshots of your drawn field when you marked it to generate a course.

**Comment by dnwalker68** [comment]
Here's a picture of the situation CP11

http://up.picr.de/46323224fu.jpg

**Reply by Tensuko** [reply]
Wow and what settings do you use?
I mean for the generator?
That lioks pretty impossible with more then 3m

**Comment by dnwalker68** [comment]
At some point this night a course was created that unfortunately cannot be used because it partially leaves the field and therefore collides with other things.

http://up.picr.de/46327214iy.jpg

**Reply by Tensuko** [reply]
It just looks like leave the field but wow did you not try to put headlands in? And again I think 10 would not fit.
